src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -42,27 +42,29 @@ public double getValue() {
     }
 
     /**
-     * Fetch a {@link Association} by its value.
+     * Fetch a {@link Association} by its exact value.
      *
      * @param value The friendliness value
      * @return The association related to the friendliness value. <code>null</code> if there is no {@link Association} with the provided value.
      * @author 086
      */
-    public static Association byValue(double value) {
+    public static Association byExactValue(final double value) {
         return Arrays.stream(values()).filter(association -> association.getValue() == value).findAny().orElseGet(null);
     }
 
     /**
-     * Fetch the {@link Association} closest to a friendliness value.
-     * <p>Edge cases: <code>0.5</code> becomes {@link Association#ALLY} and <code>-0.5</code> becomes {@link Association#ENEMY}</p>
+     * Fetch the {@link Association} convert value to association.
+     * No matter how friendly someone is as long as he is friendly he gets the ally association. The same goes for enemies.
+     * So values below 0 are enemies and above are allies.
```

Comment:
Try to make this a bit more technical - javadocs are meant to be straight to the point, specific, and most importantly, correct. We don't speak of "he"'s "she"'s, "you", etc. in javadocs.

```
Values are resolved as such:
* If the provided value is lower than the standardised neutral value, this method will return {@link Association#ENEMY}
* If the provided value is higher than the standardised neutral value, this method will return {@link Association#ALLY}
* If the provided value is equal to the standardised neutral value, this method will return {@link Association#NEUTRAL}

```

Reply:
good point